auraed/src/lib.rs

```diff
@@ -225,6 +227,44 @@ impl AuraedRuntime {
     }
 }
 
+#[derive(Hash)]
+struct CellID {
+    base: String,
+    command: String,
+    timestamp: SystemTime,
+}
+
+/// A deterministic function used to calculate a meaningful cgroup directory
+/// name from an arbitrary executable shell string.
+/// Note: The return value of this function is how we look up Aurae cells
+/// Note: from other subsystems
+fn cell_name_from_string(command: &str) -> Result<String, anyhow::Error> {
+    let now = SystemTime::now();
```

Comment:
this is not idempotent so the cell name will not be deterministic (per the comment).  or did i misunderstand _how_ deterministic you want this to be?